docs/user/ppl/cmd/stats.rst

```diff
@@ -302,6 +306,76 @@ Example::
     | 36                  | M      |
     +---------------------+--------+
 
+LIST
+----
+
+Description
+>>>>>>>>>>>
+
+Version: 3.3.0 (Calcite engine only)
+
+Usage: LIST(expr). Returns an array containing all values of the specified field from the result set, preserving both duplicates and the original order of appearance.
+All values are converted to strings for consistent handling across different data types. The results are always limited to first 100.
```

Comment:
can I increase or decrease in any way?

Reply:
List by definition is limited to top 100. The other way is to use `take` function which supports limit and explicitly convert type with `cast` function.